forem/forem
# Only show user count for large userbases clean up

## What type of PR is this? (check all applicable)


## Description

This applies PR 'Only show user count for large userbases #11021' to app/views/shared/_auth_widget.html.erb

## Related Tickets & Documents

#11021 & #11914

## QA Instructions, Screenshots, Recordings

1. Ensure you are logged out.
2. Refresh and view the auth widget displayed above the navigation links on the left sidebar, it shouldn't have the user count in there, instead, it is only displayed when the large db threshold is met from this method in application_helper.rb

'
def display_estimated_user_count?
    estimated_user_count > LARGE_USERBASE_THRESHOLD
end
'

## Added tests?

- [x] No, as they are used not the original #11021 and this is just a cleanup.

## Added to documentation?

      [Admin Guide](https://forem.gitbook.io/forem-admin-guide/)

## [optional] Are there any post deployment tasks we need to perform?

No

## [optional] What gif best describes this PR or how it makes you feel?

![counting](https://media.giphy.com/media/3tEFVAbfzzcwo/giphy.gif)
